app/controllers/api/teams.php

```diff
@@ -727,8 +727,8 @@
     ->label('audits.resource', 'team/{request.teamId}')
```

Comment:
We also need to update `->desc()` value.

Reply:
done!